woocommerce/woocommerce-ios
# REST API: Migrate orders endpoint

Closes #8550 


## Description

This PR updates all mappers related to orders endpoints: `ReportOrderTotalsMapper`, `OrderListMapper`, `OrderMapper`, `OrderNoteMapper` to parse contents without the data envelope.

All endpoints with paths `/orders`, `/notes` and `/reports/orders/totals` are enabled for REST API requests.

## Testing instructions

- Enable the feature flag `applicationPasswordAuthenticationForSiteCredentialLogin` and build the app.
- Log out of the app or skip onboarding if needed.
- On the prologue screen, select Enter your site address and input the address of a self-hosted store.
- Enter correct site credentials to log in.
- When the login succeeds, navigate to the Orders tab. Try creating, updating, loading orders. Everything should work properly.

Please feel free to test again by logging in with a WPCom account. All order-related features should work too.

## Screenshots

N/A

---
